doc/barclamp.md

```diff
@@ -220,21 +221,21 @@ this phase development will have to move to your Crowbar admin node, though.
 
 #### Chef Cookbook
 
-Location: `chef/cookbooks/mybarclamp`
+Location: `chef/cookbooks/`
 
 The best point to start barclamp development is the
 chef cookbook. Each barclamp contains
 one of these. The chef cookbook is your barclamp's core component. It contains
 the code that will do the grunt work of deploying the services, configuration
 files and runtime state handled by your barclamp. As a first step, create
-`chef/cookbooks/mybarclamp` now.
+`chef/cookbooks/` now.
 
 #### Cookbook metadata
 
 Locations:
 
-* `chef/cookbooks/mybarclamp/metadata.rb`
-* `chef/cookbooks/mybarclamp/README.md`
+* `chef/cookbooks//metadata.rb`
```

Comment:
Looks like here and in many other places you accidentally removed `mybarclamp` in this commit.